cmd/manager/main.go

```diff
@@ -107,14 +110,32 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Setup Scheme for OpenShift Routes
+	if err := routev1.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "")
+		os.Exit(1)
+	}
+
+	// Setup Scheme for OpenShift Apps group (DeploymentConfig)
+	if err := openshiftappsv1.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "")
+		os.Exit(1)
+	}
+
+	// Setup Scheme for OpenShift images group (ImageStream)
+	if err := imagev1.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "")
+		os.Exit(1)
+	}
+
 	// Setup all Controllers
 	if err := controller.AddToManager(mgr); err != nil {
 		log.Error(err, "")
 		os.Exit(1)
 	}
 
 	if err = serveCRMetrics(cfg); err != nil {
-		log.Info("Could not generate and serve custom resource metrics: ", err.Error())
+		log.Info("Could not generate and serve custom resource metrics", "error:", err.Error())
```

Comment:
Changed this to fix an error: https://github.com/operator-framework/operator-sdk/pull/1647

Reply:
+1